### PR TITLE
Make swap summary more prominent

### LINKED ISSUE #201: Make the Swap Summary be more present for the user
We have a swap summary after every swap to highlight the information if the swap was successful or not and to which page he is going to be redirected. 

For that there is a new web socket message type called `summary`.

This message type clears the page and receives an array of text to print. 

It would be nicer if it is highlighted more and maybe have some icons added (DAI or RDN).
We can discuss the protocoll what information the submitted json needs to contain.

### PR DESCRIPTION
**Pull request by manuelwedler**
Closes #201 
Makes the swap summary text bigger and shows the swapped token icon

### CODE REVIEW

File: raiden_installer/web.py [hunk 1]
```diff
@@ -375,7 +379,7 @@ def _run_swap(self, **kw):
                     redirect_url = self.reverse_url("launch", configuration_file.file_name)
                     next_page = "You are ready to launch Raiden! ..."
 
-                self._send_summary(["Congratulations! Swap Successful!", next_page])
+                self._send_summary(["Congratulations! Swap Successful!", next_page], icon=token_ticker)
```
Comment (fredo) [inline raiden_installer/web.py:382]: Should we handle the Error Case as well with an icon (Line 393) ? One suggestion: Lets introduce a boolean `successful` which indicates if the Transaction went through. In the Error case we could add like a "splash" symbol indicating better that something went wrong

Comment (fredo) [reply raiden_installer/web.py:382]: And also, do you want to show the swapped token or the upcoming one?

Comment (manuelwedler) [reply raiden_installer/web.py:382]: I want to show the swapped token.

Comment (manuelwedler) [reply raiden_installer/web.py:382]: We could also just set the icon variable to `error` in the error case and add such an image. That way we would not need to have another argument that needs to be checked.

Comment (fredo) [reply raiden_installer/web.py:382]: also good idea

Comment (fredo) [reply raiden_installer/web.py:382]: lets do it this way

**Pull request merged**